rca/people/formatters.py

```diff
@@ -0,0 +1,23 @@
+def format_research_highlights(pages):
+    """Function for formatting related projects to the correct
+    structure for the gallery template
+    Returns:
+        List
+    """
+    items = []
+    for page in pages:
```

Comment:
would `select_related('related_school_pages')` or `prefetch_related('related_school_pages')` help reduce queries?

Reply:
I don't think I can do that, the reason being is that the research pages [here](https://github.com/torchbox/rca-wagtail-2019/blob/cf8b80e814a69d6c1d2068b5300f7494912661b5/rca/people/models.py#L338-L339) come from a generator (yield) in https://github.com/torchbox/rca-wagtail-2019/blob/cf8b80e814a69d6c1d2068b5300f7494912661b5/rca/people/utils.py#L1-L46